gtsfm/loader/yfcc_imb_loader.py

```diff
@@ -129,7 +125,10 @@ def is_valid_pair(self, idx1: int, idx2: int) -> bool:
         Returns:
             validation result.
         """
-        return (idx1, idx2) in self._image_pairs
+        if not super().is_valid_pair(idx1, idx2):
```

Comment:
will this affect the results across their benchmark in any way? i.e. how is their GT provided?

Reply:
No it should not. We are just being consistent in how we store the pairs internally in this class.